torch/csrc/api/src/nn/modules/linear.cpp

```diff
@@ -56,6 +56,20 @@ Tensor LinearImpl::forward(const Tensor& input) {
 
 // ============================================================================
 
+FlattenImpl::FlattenImpl(const FlattenOptions& options_) : options(options_) {}
+
+void FlattenImpl::reset() {}
+
+void FlattenImpl::pretty_print(std::ostream& stream) const {
+  stream << "torch::nn::Flatten(start_dim=" << options.start_dim() << ", end_dim=" << options.end_dim() << ")";
+}
+
+Tensor FlattenImpl::forward(const Tensor& input) {
+  return torch::flatten(input, options.start_dim(), options.end_dim());
```

Comment:
We might be able to call
```suggestion
  return input.flatten(input, options.start_dim(), options.end_dim());
```
to match the Python version even better :D